src/patches/win32_gdi_point_patch.py

```
@@ -0,0 +1,43 @@
+        point_ptr = ctypes.POINTER(wintypes.POINT)
+        user32.GetCursorPos.argtypes = [point_ptr]
+        user32.UpdateLayeredWindow.argtypes = [
+            ctypes.c_void_p, ctypes.c_void_p, point_ptr,
+            ctypes.POINTER(win32_gdi.SIZE), ctypes.c_void_p, point_ptr,
+            wintypes.DWORD, ctypes.POINTER(win32_gdi.BLENDFUNCTION), wintypes.DWORD,
+        ]
+        gdi32.MoveToEx.argtypes = [ctypes.c_void_p, ctypes.c_int, ctypes.c_int, point_ptr]
+        _PATCH_INSTALLED = True
+    except Exception:
+        # 若 ok 库版本/结构变化导致补丁失败，静默跳过，不影响启动
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)